packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -137,12 +137,13 @@ export class MarketOperationUtils {
         const _opts = { ...DEFAULT_GET_MARKET_ORDERS_OPTS, ...opts };
         const [makerToken, takerToken] = getNativeOrderTokens(nativeOrders[0]);
         const sampleAmounts = getSampleAmounts(takerAmount, _opts.numSamples, _opts.sampleDistributionBase);
+
         const requestFilters = new SourceFilters().exclude(_opts.excludedSources).include(_opts.includedSources);
-        // We don't exclude from the fee sources as we always want to be able to get a price
-        // of taker to Eth or maker to Eth, especially for MultiHop
-        const feeSourceFilters = this._feeSources;
         const quoteSourceFilters = this._sellSources.merge(requestFilters);
 
+        const feeRequestFilters = new SourceFilters().exclude(_opts.excludedFeeSources);
+        const feeSourceFilters = this._feeSources.merge(feeRequestFilters);
```

Comment:
Isn't this the same as just..?
```suggestion
        const feeSourceFilters = this._feeSources.exclude(_opts.excludedFeeSources);
```

Reply:
done